Don't look for the framework path, we build Python the UNIX way for
Gentoo Prefix

--- a/engine/build.jam
+++ b/engine/build.jam
@@ -82,11 +82,6 @@
         }
         --python-lib = $(--python-lib[1]) ;
     }
-    else if $(OS) = MACOSX
-    {
-        --python-include = [ .path $(python-location) Headers ] ;
-        --python-lib = $(python-location) Python ;
-    }
     else
     {
         --python-include = ;
